src/parsers/DirSearchParser.java

```diff
@@ -37,7 +37,9 @@ URL parseDirectory(String urlString) throws MalformedURLException {
                     if(radioButton.getText().equalsIgnoreCase("simple-report")){
                         return new URL(urlString);
                     } else if(radioButton.getText().equalsIgnoreCase("plain-text-report")){
-                        return new URL(urlString.split(" {3}")[2]);
+                        if (urlString.matches(".*   .*  .*")) {
```

Comment:
Does this work? Isn't `.*   .*  .*` the same as `.*`

Reply:
Adjusted. Using "^\d{3}.*" now